custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -164,16 +186,8 @@ def get_prompt_column_name(self):
             )
             return DEFAULT_PROMPT_COLUMN_NAME
 
-        try:
-            deployment = dr.Deployment.get(self._params["deployment_id"])
-            return deployment.model["prompt"]
-        except Exception:
-            logger.warning(
```

Comment:
I think it would be helpful to still have a warning log somewhere which informs user that we may fallback to default prompt column name

Reply:
ok, i will add that